src/mudlet.pro

```diff
@@ -248,7 +248,10 @@ isEmpty( MAIN_BUILD_SYSTEM_TEST ) | !equals( MAIN_BUILD_SYSTEM_TEST, "NO" ) {
 # the map file gets requested and when such a request gets actioned (or
 # rejected because the profile is being loaded):
 # DEFINES+=DEBUG_MAPAUTOSAVE
-
+#
+# * Produce all the time the surprise that normally will only occur on the first
+# day of the fourth month of the Gregorian calendar year:
+DEFINES+=DEBUG_EASTER_EGGS
```

Comment:
```suggestion
// DEFINES+=DEBUG_EASTER_EGGS
```

Reply:
This also isn't a C++ file - you need to use # not //...!